src/jit.cpp

```diff
@@ -150,7 +150,30 @@ void anydsl_link(const char* lib) {
 }
 
 int32_t anydsl_compile(const char* program, uint32_t size, uint32_t opt) {
-    return jit().compile(program, size, opt);
+    return jit().compile(program, size, opt, thorin::LogLevel::Warn);
+}
+
+int32_t anydsl_compile2(const char* program, uint32_t size, uint32_t opt, uint32_t loglevel) {
+    thorin::LogLevel level = thorin::LogLevel::Warn;
+    switch(loglevel) {
```

Comment:
Why have a switch-case here ? You could use the enum directly, or cast an int to the enum if you need FFI

Reply:
We definitely could do that. Just don't want to expose the thorin headers to public. This would require switching the current order (but thats no problem). 
I personally don't like casting integers directly to enums due to undefined behavior if the integer is out of range. This is no performance related block, so here I think a switch cast is ok.

Reply:
It's not about performance, but rather making the code clear and concise. I'm not concerned about providing a log level out of range, we don't do input validation on any of the other fields either. These 19 lines could be just one:
```c++
thorin::LogLevel level = static_cast<thorin::LogLevel>(loglevel <= 4 ? loglevel : 1);
```

Reply:
Ohh, like the latter one. Will update it real quick